src/res/failcode.ja.json

```diff
@@ -575,11 +575,13 @@
     "nmTentGt.tents": "木の数よりもテントの数が多い部分があります。",
     "nmTentLt.tents": "木の数よりもテントの数が少ない部分があります。",
     "nmTentNone.tents": "テントが隣り合っていない木があります。",
+    "nmTouchNe.tslither": "この数は、セルのエッジ/頂点への個別の訪問の数と同じではありません",
```

Comment:
I've recently made Japanese Failcodes optional (in the `master` branch), so these Google Translated ones aren't needed anymore.

Reply:
Removed this line